### PR TITLE
Updated README.md to be more concise.

### PR DESCRIPTION
**Pull request by brandonboesch**
Made a few updates to make the documentation more concise for the user.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,10 +110,10 @@ The next step is to build an application you can combine with your bootloader to
 
 1. Connect the SD card to your computer.
 1. Copy the application binary (``<program name>_application.bin``, we defined it earlier as ``mbed-os-example-blinky_application.bin``) to the root of the SD card.
-1. Remove the SD card from your PC, and plug it into the mbed board.
+1. Remove the SD card from your PC. Ensure the CI test shield is attached to your device, and then plug the SD card into the CI test shield.
 1. Press the reset button to start the firmware update.
 
-If a terminal is open, the following prints:
+If a terminal is open, the following should print out on baud rate of 115200:
```
Comment (bridadan) [inline README.md:116]: I don't believe this is correct, it should be 9600. What board were you testing with?

File: README.md [hunk 2]
```diff
@@ -110,10 +110,10 @@ The next step is to build an application you can combine with your bootloader to
 
 1. Connect the SD card to your computer.
 1. Copy the application binary (``<program name>_application.bin``, we defined it earlier as ``mbed-os-example-blinky_application.bin``) to the root of the SD card.
-1. Remove the SD card from your PC, and plug it into the mbed board.
+1. Remove the SD card from your PC. Ensure the CI test shield is attached to your device, and then plug the SD card into the CI test shield.
```
Comment (bridadan) [inline README.md:113]: The CI test shield isn't actually necessary here. Some boards have SD card slots built in, others can be added via various SD card shields/breakouts. I'd recommend leaving this line as is.

File: README.md [hunk 3]
```diff
@@ -76,15 +76,15 @@ Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 
 It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
 
-When the build succeeds, you have created a bootloader for your target. This example defines the application file to be located at `/sd/mbed-os-example-blinky_application.bin` in the application config file ``mbed_app.json``. This is the application binary bootloader flashes and then jumps to.
+When the build succeeds, you have created a bootloader for your target. This example defines the application file to be located at `/sd/mbed-os-example-blinky_application.bin` in the bootloader's ``main.cpp``. This is the application binary that bootloader flashes and then jumps to. If you are using a different application, ensure that ``main.cpp`` is pointing to the correct path for the application.
```
Comment (bridadan) [inline README.md:79]: Probably should be "that the bootloader flashes and then jumps to."

**Review by bridadan — COMMENTED** [review]
Maybe we should integrate your changes into my existing PR? I've touched quite a few of the same paragraphs that you have.

**Comment by c1728p9** [comment]
@brandonboesch can you merge your changes into #19

**Comment by 0xc0170** [comment]
As this was not merged into the another one, can you @brandonboesch please rebase t oresolve conflicts ? thanks

**Comment by bridadan** [comment]
@brandonboesch we added all of this info in pr #22, do you think its ok to close this?

**Pull request closed without merging**